client/src/hooks/graphql/use-with-import-export.tsx

```diff
@@ -32,7 +35,8 @@ export function useWithImportExport(accessToken: string): UseWithImportExport {
   const [importedJson, setImportJson] = useState<MentorExportJson>();
   const [importPreview, setImportPreview] = useState<MentorImportPreview>();
   const [isUpdating, setIsUpdating] = useState(false);
-  const { mentor, loadMentor } = useWithMentor();
+  const mentor = useActiveMentor((state) => state.data);
```

Comment:
make sure to only select what you're using as in other examples (didn't check, maybe in this case you really use the whole mentor?)